routes/web.php

```diff
@@ -60,6 +61,14 @@
 Route::get('/task/comment/{id}',[TaskCommentController::class,'findTaskCommentByIdTest']);
 
 //Resource route
+Route::get('/taskresource', [App\Http\Controllers\TaskDemoController::class, 'resource']);
```

Comment:
Please we are not doing anything with the Demo controller its just a demo controller